cmd/cluster-init/config_io.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"io/ioutil"
+	"sigs.k8s.io/yaml"
+)
+
+func loadConfig(filename string, o interface{}) {
```

Comment:
nit: Go leans very heavily into being verbose. While it may look DRY to write a function like this, it is not, as the inputs to all of the calls are parameterized. Furthermore, `o interface{}` is the Go equivalent of `void*` or `Object` and circumvents the type system, which is not desirable.

Reply:
This is listed as a "nit" but I really would like to learn the go way to do this. The `yaml.Unmarshal` function takes `o interface{}` as a parameter so I was mimicking the signature there. Do you suggest that this all be inlined for _each_ config that I am utilizing?

Reply:
Yep! That's the verbosity cost of Go. Seems crazy, but the general thought is - so what? This code is written once. Hiding *how* you serialize and write a file only makes the callsite harder to read, and there's no real cost after write-time for `ioutil.ReadFile()` followed by `yaml.Unmarshal`, so the benefit to read is where the language heads.